pkg/users/users.js

```diff
@@ -71,7 +71,7 @@ function AccountsPage() {
 
     useInit(() => {
         const debouncedGetLogins = debounce(100, () => {
-            getLogins().then(setDetails);
+            getLogins().then((logins) => setDetails(logins.filter((login) => !!login)));
```

Comment:
This is a bit too late for my taste -- empty logins should rather be filtered out inside of getLogins() in the output parsing of the respective program.

Do you have an example `lastlog` or `w` or `passwd -S --all` output which causes that? I'd like to put that into our integration test to actually cover and verify this update.

Reply:
I've fixed this on the catch line at 212 - not returning anything from that was causing the undefined to show up. This at least fixes the issues on systems with non-coreutils date